Makefile

```diff
@@ -61,7 +61,7 @@ IMG ?= $(DEFAULT_IMG)
 ENVTEST_K8S_VERSION = 1.25.0
 GINKGO ?= $(LOCALBIN)/ginkgo
 
-PROCS?=$(shell expr $(shell nproc --ignore 2) / 2)
+PROCS ?=$(shell expr $(shell nproc --ignore 2) / 4)
```

Comment:
looks like reducing the parallel execution makes the job stable enough.
It never failed after hours of local testing, and didn't get any new failure in CI after tuning the `--procs` value.
@gibizer unless you have a better idea I would go with this.